packages/@react-spectrum/provider/src/Provider.tsx

```diff
@@ -49,6 +50,7 @@ function Provider(props: ProviderProps, ref: DOMRef<HTMLDivElement>) {
     colorScheme = usePrevColorScheme ? prevColorScheme : autoColorScheme,
     scale = prevContext ? prevContext.scale : autoScale,
     locale = prevContext ? prevLocale : null,
+    breakpoints = prevContext ? prevBreakpoints : {S: 380, M: 768, L: 1024},
```

Comment:
Where did the defaults come from? not questioning them, just want a comment here explaining.

Reply:
They are from https://github.com/adobe/react-spectrum/issues/770
which is a temporary value. I think we need to define more appropriate defaults.